include/gepetto/viewer/urdf-parser.h

```diff
@@ -12,6 +12,30 @@
 #include <gepetto/viewer/group-node.h>
 #include <gepetto/viewer/leaf-node-collada.h>
 
+#ifndef URDFDOM_BOOST_SHARED_PTR
+#include <memory>
+namespace graphics {
+  namespace urdfParser {
+    using std::shared_ptr;
+    using std::weak_ptr;
+    using std::static_pointer_cast;
+    using std::dynamic_pointer_cast;
+  }
+}
+#else
+#include <boost/shared_ptr.hpp>
+#include <boost/weak_ptr.hpp>
+namespace graphics {
+  namespace urdfParser {
+    using boost::shared_ptr;
+    using boost::weak_ptr;
+    using boost::static_pointer_cast;
+    using boost::dynamic_pointer_cast;
+  }
+}
+#endif
+
+
```

Comment:
This should go in the source file.

In the header file, it can create a weird scenario where `graphics::shared_ptr` is from `std` and `graphics::urdfParser::shared_ptr` is from `boost`. And this will be in all files which includes this file.

I agree that the namespace `urdfParser` should be sufficient to avoid name collisions but I would rather avoid the risk of collision as it is easily to avoid.